templates/webshop/search-results.jinja

```diff
@@ -34,16 +34,15 @@
                   {% endfor %}
                 </ul>
               {% if value['terms']|length > 5 %}
-                  <div class="more btn btn-default btn-sm toggle-show-more">
-                    <span class="hide-on-more">
-                      {% trans %}Show More{% endtrans %}
-                      <span class="glyphicon glyphicon-plus"></span>
-                    </span>
-                    <span class="show-on-more">
-                      {% trans %}Show Less{% endtrans %}
-                      <span class="glyphicon glyphicon-minus"></span>
-                    </span>
-                  </div>
+                <div class="more btn btn-default btn-sm toggle-show-more">
+                  <span class="hide-on-more">
+                    {% set remaining_list = 5 %}
```

Comment:
move this line to start and use this variable in whole file. call this variable `facet_show_limit`